data/containers/Dockerfile

```diff
@@ -13,5 +13,8 @@ COPY index.html .
 # tell the port number the container should expose
 EXPOSE 5000
 
+# Set experimental variable
+ENV WORLD_VAR Arda
```

Comment:
where is this var used now? The only code using it was `build_container_image` which is now gone.

Reply:
Sorry Jose, I by mistake removed `lib/containers/utils.pm` - please check that file - it's used there.